drafts/0007-RFC-BAG-OPERATORS.md

```diff
@@ -4,26 +4,26 @@
 # Summary
 [summary]: #summary
 
-This RFC defines the PartiQL specification for the operators: UNION, INTERSECT, and EXCEPT.
+This RFC defines the PartiQL specification for the operators: UNION, INTERSECT, and EXCEPT. The behavior of each operator conforms to the SQL standard in strict typing mode. In the absence of schemas, each operator is equivalent to the mathematical definitions of the respective multi-set operator where member equality is consistent with the equality in the *GROUP BY* clause.
 
 # Motivation
 [motivation]: #motivation
 
-The operators UNION, INTERSECT, and EXCEPT have been defined since SQL-92 specifications, yet these are missing from the PartiQL spec.
-In SQL, these operators are the typical bag (set if DISTINCT) union, intersect, and difference respectively for *union-compatible* relations. Unlike SQL, PartiQL is schemaless and therefore these operators are valid for arbitrary expressions — including a SFW block.
+The operators UNION, INTERSECT, and EXCEPT have been defined since SQL-92 specifications, yet these are missing from the PartiQL spec. In SQL, these operators are the typical bag (set if DISTINCT) union, intersect, and difference respectively for *union-compatible* relations. In the presence of schemas, the PartiQL set operators are compatible with the ISO SQL specification for set operators; however, PartiQL extends these operators to allow for combining arbitrary values such as bags of heterogenous tuples. SQL compatbility is discussed in the following section, and a detailed look at SQL set operators can be found in Appendix A.
```

Comment:
Where is the term "union-compatible" coming from?  Just an interesting term given intersection and except are different set operations.